### PR TITLE
Updated fh-forms from 1.16.6 -> 1.16.7

### PR DESCRIPTION
**Pull request by davidkirwan**
JIRA: https://issues.jboss.org/browse/RHMAP-20186
Updated via: https://github.com/feedhenry/fh-forms/pull/46

### CODE REVIEW

**Comment by davidkirwan** [comment]
@camilamacedo86 @davidffrench @grdryn is there anything else required in order to have my fh-forms change take effect in fh-mbaas?

**Comment by camilamacedo86** [comment]
@davidkirwan it will not work directly. We need to publish the fh-forms first. (https://issues.jboss.org/browse/RHMAP-20414)

**Comment by davidffrench** [comment]
@davidkirwan as Camila suggest, fh-forms will need to be published to npm. For this repo, alongside updating the package.json, the shrinkwrap will need to be re generated and the version of fh-mbaas bumped.

**Procedure for publishing to npm:** https://github.com/fheng/help/blob/7c0812f31049dec7ecedaf9cfc7487cc653073a8/developer_guides/engineering_wiki/Deployment-for-client-node-modules.md

**Comment by davidkirwan** [comment]
Perfect thanks @camilamacedo86 I'll follow these steps and ping you again when done!

**Comment by matskiv** [comment]
New version has been added to **devDependencies**, and shrinkwrap was not updated.

File: package.json [hunk 1]
```diff
@@ -54,6 +54,7 @@
   },
   "devDependencies": {
     "deep-equal": "~1.0.1",
+    "fh-forms": "1.16.7",
```
Comment (camilamacedo86) [inline package.json]: Hi @davidkirwan,

Following the steps to update the dep here. 
* Run `npm install --save --save-exact fh-forms@1.16.7` to update the dep in the prod and not dev section
* Check if the dep was updated in the file [npm-shrinkwrap](https://github.com/feedhenry/fh-mbaas/blob/master/npm-shrinkwrap.json#L1181), if not delete:
** Delete the file and the node_modules dir 
** Run `npm install`
** Run `npm shrinkwrap` in the root dir of this project
* Then commit the changes 
* Also, add a new commit to bump the version of the fh-mbaas api.

Comment (davidkirwan) [reply package.json]: Ran the first command and ended up with this inside the npm-shrinkwrap.json file:

```
    "fh-forms": {
      "version": "1.16.6",
      "from": "fh-forms@1.16.6",
      "resolved": "https://registry.npmjs.org/fh-forms/-/fh-forms-1.16.6.tgz",
```

Comment (davidkirwan) [reply package.json]: Will retry by deleting this npm-shrinkwrap.json file, and the node_modules then following on with the ```npm install``` and ```npm shrinkwrap```

**Comment by davidkirwan** [comment]
@camilamacedo86 @matskiv @matskiv @grdryn updated the files as per instructions, does this look ok now?

File: package.json (hunk 1 above)
Comment (davidkirwan) [reply package.json]: This appeared to have worked.

Comment (grdryn) [reply package.json]: This should be removed from the devDependencies section, given that it's in the dependencies section now.

File: licenses/MOMENT_MIT.TXT [hunk 2]
```diff
@@ -20,3 +20,4 @@ HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
 WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
 FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
 OTHER DEALINGS IN THE SOFTWARE.
+INGS IN THE SOFTWARE.
```
Comment (davidffrench) [inline licenses/MOMENT_MIT.TXT]: @davidkirwan unfortunately the license-reporter tool is not perfect sometimes. This line can be removed as it shouldn't be here.

**Comment by camilamacedo86** [comment]
@davidkirwan LGTB for me 👍 Good job!

**Pull request merged**